provider/gpkg/gpkg_register.go

```diff
@@ -298,7 +304,8 @@ func NewTileProvider(config map[string]interface{}) (provider.Tiler, error) {
 			// Set bounds & zoom params to include all layers
 			// Bounds checks need params: maxx, minx, maxy, miny
 			// TODO(arolek): this assumes WGS84. should be more flexible
-			customSQL = replaceTokens(customSQL, 0, geom.BoundingBox{{180.0, 85.0511}, {-180.0, -85.0511}})
+			wgs84BB := geom.NewExtent([2]float64{180.0, 85.0551}, [2]float64{-180.0, -85.0511})
```

Comment:
maybe use: 

```go
geom.NewExtent([2]float64{tegola.WGS84Bounds[3], tegola.WGS84Bounds[2]}, [2]float64{tegola.WGS84Bounds[0], tegola.WGS84Bounds[1]})
```

or we could change the tegola.WGS84Bounds to a geom.Extent{}.